design/resources.go

```diff
@@ -379,3 +379,33 @@ var _ = a.Resource("trackerquery", func() {
 	})
 
 })
+
+var _ = a.Resource("search", func() {
+	a.BasePath("/search")
+
+	a.Action("show", func() {
+		a.Routing(
+			a.GET(""),
+		)
+		a.Description("Search by ID, URL, full text capability")
+		a.Params(func() {
+			a.Param("q", d.String,
+				`Following are valid input for seach query
+				1) id:100 :- Look for work item hainvg id 100
+				2) url:http://demo.almighty.io/details/500 :- Search on WI having id 500 and check 
+					if this URL is mentioned in searchable columns of work item
+				3) simple keywords seperated by space :- Search in Work Items based on these keywords.`)
+			a.Param("page[offset]", d.Integer, "Paging in the format <start>,<limit>")
```

Comment:
See description https://github.com/almighty/almighty-core/issues/428

No, Limit is an Integer.